docs/source/integrations/cvat.rst

```diff
@@ -1957,6 +1964,58 @@ attributes between annotation runs.
    :alt: cvat-occ-widget
    :align: center
 
+
+.. _cvat-destination-field:
```

Comment:
nit: stick to just one blank line between blocks in all cases (see rest of file).

I wish that our pre-commit hook would automatically enforce this like it does for Python, but unfortunately it doesn't. Goal is just that everyone use the same style to minimize diffs.